IOHexperimenter/Extension.md

```diff
@@ -93,4 +93,68 @@ public:
 };
 ```
 
-Please check [this example](https://github.com/IOHprofiler/IOHexperimenter/blob/759750759331fff1243ef9e121209cde450b9726/example/problem_example.h#L51) for adding continuous problems in this manner.
\ No newline at end of file
+Please check [this example](https://github.com/IOHprofiler/IOHexperimenter/blob/759750759331fff1243ef9e121209cde450b9726/example/problem_example.h#L51) for adding continuous problems in this manner.
```

Comment:
@jacobdenobel Could you please take a look at this section to check if this is fully up-to-date with the recent changes?